lib/commands/order/get_all_orders_command.rb

```diff
@@ -15,7 +15,11 @@ def prepare(data)
   private
 
   def orders
-    Order.all.map { |order| "#{order.user_name}: #{order.lunch}" if !order.lunch.nil?}.compact.sort
+    orders_of_the_week = Order.all(:date => (Days.from_monday_to_friday))
+
+    orders_of_the_week.map {
+      |order| "#{order.user_name}: #{order.lunch}" if !order.lunch.nil?
```

Comment:
It's more idiomatic to put the block parameter on the same line the `do` or `{` of the block is.